softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/for_authorization/Messages.java

```diff
@@ -0,0 +1,91 @@
+package com.softgroup.common.dao.api.entities.for_authorization;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "messages")
+public class Messages {
```

Comment:
@AlexBort All entities must have Entity suffix. So this must be named MessageEntity